hack/local-development/remote-garden/open-gardener-tunnels

```diff
@@ -263,11 +277,38 @@ echo "Applying gardener services..."
 applyGardenerServices
 
 echo "Starting quic clients..."
-docker run -d --name $APISERVER_SERVICE_NAME-quic-client -l "$LABEL" --network host --rm -v "$CERTS_DIR":/certs $QUIC_CLIENT_IMAGE \
+
+# the DNS name 'host.docker.internal' is only available for containers run by docker
+# for desktop on Mac / Windows - otherwise use localhost.
+# Check: desktop on Mac / Windows has not docker0 interface on the host
+# should be run on the actual host (important when running docker over ssh)
+hostname="host.docker.internal"
+if ip addr show docker0 > /dev/null 2>&1; then
+    hostname="localhost"
+fi
+
+docker run \
+  -d \
+  --name $APISERVER_SERVICE_NAME-quic-client \
+  -l "$LABEL" \
+  --network host  \
+  -v "$CERTS_DIR":/certs \
+  $QUIC_CLIENT_IMAGE \
   --server="$loadbalancerHostname:$APISERVER_LB_PORT" \
-  --upstream="host.docker.internal:$API_SERVER_SECURE_PORT" \
-  --ca-file=/certs/$CA_NAME.crt --cert-file=/certs/$CLIENT_NAME.crt --cert-key=/certs/$CLIENT_NAME.key
-docker run -d --name $ADMISSION_CONTROLLER_SERVICE_NAME-quic-client -l "$LABEL" --network host --rm -v "$CERTS_DIR":/certs $QUIC_CLIENT_IMAGE \
+  --upstream="$hostname:$API_SERVER_SECURE_PORT" \
+  --ca-file=/certs/$CA_NAME.crt \
+  --cert-file=/certs/$CLIENT_NAME.crt  \
+  --cert-key=/certs/$CLIENT_NAME.key
```

Comment:
Is there any particular reason you removed the `--rm` flag? I think it should better stay to ensure we don't leave hanging containers, also it's used in similar `docker run` commands in `local-garden`

Reply:
Thanks - you are right. I removed the flag for testing to see container logs but forgot to add it again.

Reply:
added it again